src/ui/public/filter_bar/put_filter.js

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
+
```

Comment:
can you explain this as well ? why filter_bar_click_handler was not useful ?

i would prefer this to be an object with methods i can call on it (for the sake of plugin API we are talking about)

so instead of me having to know all the paths to functions in our project
```
import putFilter from 'ui/filter_bar/put_filter';
import removeFilter from 'ui/filter_bar/remove_filter';
import clickHandler from 'ui/filter_bar/filter_bar_click_handler';
```

i would prefer something like
```
import filterHandler from 'ui/filter_bar/filter_handler';
filterHandler.click(); filterHandler.put(), filterHandler.remove();
```

or in the future even
```
 import { filterHandler, searchHandler, timepickerHandler } from 'ui/plugin_api';
```


maybe it would be cool if we would do the first step here ?